services/minespace-web/src/components/Forms/projects/projectSummary/Agent.tsx

```diff
@@ -16,21 +16,20 @@ import {
 } from "@mds/common/redux/utils/Validate";
 import { getDropdownProvinceOptions } from "@mds/common/redux/selectors/staticContentSelectors";
 
-export const Agent: FC = () => {
+interface AgentProps {
+  countryOptions: any;
+}
+
+export const Agent: FC<AgentProps> = (props) => {
   const dispatch = useDispatch();
   const formValues = useSelector(getFormValues(FORM.ADD_EDIT_PROJECT_SUMMARY));
   const { agent = {}, is_agent = false } = formValues;
   const { party_type_code, address = {} } = agent ?? {};
   const { address_type_code, sub_division_code } = address ?? {};
   const isInternational = address_type_code === "INT";
-
-  const provinceOptions = useSelector(getDropdownProvinceOptions);
   // currently no endpoints, etc, for address_type_code
-  const countryOptions = [
-    { value: "CAN", label: "Canada" },
-    { value: "USA", label: "United States" },
-    { value: "INT", label: "International" },
-  ];
+  const { countryOptions } = props;
```

Comment:
hmm I'm thinking this would make sense to put in constants or enums instead of passing as props.

Reply:
CountryOptions has been moved to the strings file as a constant now.